### PR TITLE
Swagger for transferring subscription within a commerce root (Do not merge)

### PR DESCRIPTION
**Pull request by jetts**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 1]
```diff
@@ -384,6 +384,174 @@
...
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}/billingSubscriptions/{subscriptionId}/transfer/operationResults/{operationId}": {
```
Comment (ravbhatnagar) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: you dont need these.

**Comment by ravbhatnagar** [comment]
Looks good. Just one note- you dont need the APIs for tracking the status of long running operations in swagger. it comes from the header. 
Other than this, looks fine. Signing off from ARM side.

**Comment by sarangan12** [comment]
Please fix the erros reported in https://travis-ci.org/Azure/azure-rest-api-specs/jobs/466790033

**Comment by sarangan12** [comment]
@jetts You have added "Do not merge" in the title. Is it still valid?

**Comment by sarangan12** [comment]
@veronicagg I am assigning this to you since I will be OOF for the next 2 weeks

**Comment by veronicagg** [comment]
reassigning to @sarangan12 to pick up when he's back, as I will be OOF too. Please expect us to pick up reviews again starting Jan 2nd, as most reviewers are out during holidays. Happy holidays!

**Comment by sarangan12** [comment]
@jetts You have added "Do not merge" in the title. Is it still valid? If there is no movement in this PR in the next 2 days, I will be closing this without merge

**Pull request closed without merging**